cmd/sink-worker/main.go

```diff
@@ -68,18 +68,21 @@ func main() {
 
 	err := v.ReadInConfig()
 	if err != nil && !errors.As(err, &viper.ConfigFileNotFoundError{}) {
-		panic(err)
+		slog.Error("failed to read configuration", slog.String("error", err.Error()))
```

Comment:
Slog isn't configured at this point yet, so this is going to use the default logger and not adhere to the config: https://go.dev/play/p/BtOY4FYD-6B

Using `panic` was a conscious decision to signal something is seriously broken.

Reply:
Alright. Reverted this part of the change.